USAGE.md

```diff
@@ -327,6 +327,25 @@ Potential spelling errors will be printed in the continuous integration log alon
 Words in `build/assets/custom-dictionary.txt` are ignored during spellchecking.
 Spellchecking is currently only supported for English language manuscripts.
 
+## AI-assisted authoring
+
+The workflow `ai-revision` is available to assist authors in writing their manuscripts.
```

Comment:
I think this might work:

```suggestion
The workflow [`ai-revision`](.github/workflows/ai-revision.yaml) is available to assist authors in writing their manuscripts.
```